wicket/src/ui/panes/rack_setup.rs

```diff
@@ -791,6 +792,30 @@ fn rss_config_text<'a>(
             .map(|ip| plain_list_item(ip.to_string()))
             .collect(),
     );
+
+    // Add the allowlist for connecting to user-facing rack services.
+    let allowed_source_ip_spans = match &insensitive.allowed_source_ips {
+        None | Some(AllowedSourceIps::Any) => {
+            vec![plain_list_item(String::from("Any"))]
+        }
+        Some(AllowedSourceIps::List(list)) => list
+            .iter()
+            .map(|net| {
+                let as_str = if net.prefix() == 32 || net.prefix() == 128 {
```

Comment:
Tiny nit - I think this would be wrong if we got an Ipv6 addr with a `/32`, right? Could we check for `net.first_address() == net.last_address()` instead?

Reply:
Nice catch, thank you.

Reply:
Done in 9fd8772e4